src/main/java/com/labs/introtoprogramming/lab4/image/io/ImageReader.java

```diff
@@ -2,8 +2,10 @@
 
 import com.labs.introtoprogramming.lab4.image.RGBImage;
 
+import java.io.IOException;
+
 public interface ImageReader {
 
-  public RGBImage read();
+  public RGBImage read() throws IOException;
```

Comment:
Ooops, `public` is redundant here, so there is no need for it. Sorry for my previous comment.